src/go/rdctl/cmd/snapshotList.go

```diff
@@ -33,35 +36,55 @@ var snapshotListCmd = &cobra.Command{
 	Use:     "list",
 	Aliases: []string{"ls"},
 	Short:   "List snapshots",
+	Args:    cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return listSnapshot(cmd, args)
+		return listSnapshot()
 	},
 }
 
 func init() {
 	snapshotCmd.AddCommand(snapshotListCmd)
+	snapshotListCmd.Flags().BoolVar(&outputJsonFormat, "json", false, "output json format")
 }
 
-func listSnapshot(cmd *cobra.Command, args []string) error {
-	paths, err := paths.GetPaths()
+func listSnapshot() error {
+	snapshotPaths, err := paths.GetPaths()
```

Comment:
I find the name misleading; I would expect that `snapshotPaths` would be set to `paths.GetPaths().Snapshots` (and be called `snapshotsPath` instead of `snapshotPaths`).

Since it is just a `paths.Paths` object, I think the previous name `paths` works better.

Reply:
Adam and I were discussing the use of `paths` as a name, and concluded it would be better to import the `paths` module as `p` and use `paths` as a local variable name.